src/Lithnet.GoogleApps/UserRequestFactory.cs

```diff
@@ -72,7 +75,7 @@ public IEnumerable<User> GetUsers(string customerID, string fields, string query
                 {
                     request.PageToken = token;
 
-                    UserList pageResults = request.ExecuteWithRetryOnBackoff();
+                    UserList pageResults = request.ExecuteWithRetry(RetryEvents.Backoff | RetryEvents.Timeout, ApiExtensions.RetryCount);
```

Comment:
The ApiExtensions.RetryCount parameter is not needed here, as the appropriate overload of ExecuteWithRetry will default to this parameter

Reply:
My bad, I mistook the consumeTokens for the retrycount. Changing.